install_files/ansible-base/roles/common/defaults/main.yml

```diff
@@ -49,3 +49,4 @@ unused_packages:
   - libiw30
   - wireless-tools
   - wpasupplicant
+  - snapd
```

Comment:
We have config tests for package absence, that should be updated to confirm snapd is not present (on any system).